.github/workflows/distribute.yml

```
@@ -0,0 +1,61 @@
+
+jobs:
+  distribute:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.branch }}
+          token: ${{ secrets.GITHUB_TOKEN }}
+          fetch-depth: 0
+
+      - uses: ./.github/actions/build-configuration
```

actionlint (1.7.11)

[error] description is required in metadata of "Build Configuration" action at "/home/jailuser/git/.github/actions/build-configuration/action.yml"

(action)